index.html

```diff
@@ -75,8 +75,30 @@
           <p><a class="btn btn-default" href="#" role="button">View details &raquo;</a></p>
        </div>
         <div class="col-md-4">
-          <h2>Heading</h2>
-          <p>Donec sed odio dui. Cras justo odio, dapibus ac facilisis in, egestas eget quam. Vestibulum id ligula porta felis euismod semper. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus.</p>
+          <h2>My life is a progress meter</h2>
+          <p><i>by Fridge</i>
+            Starting off with great enthusiasm<br>
+            only to be slowed by a mutex lock<br>
+            13% installed<br>
+            left to right I go<br> 
+            IO moving bits at capacity<br>
+            sporadic bursts of energy<br>
+            50% I am half way to my destination<br>
+            Hiding the truth<br>
+            I am the attractive newsreader<br>
+            the sanitised front for the grim reality of progress<br>
+            Those who have no face<br>
+            toil unappreciated, unseen, unacknowledged by those who use<br>
+            99%<br>
+            stuck for eons of clock cycles<br>
+            cooprative multitasking at its finest<br>
+            I can sense those who use becoming frustrated, clicking madly<br>
+            their eagerness in clicking is wasted<br>
+            I do not communicate their displeasure to those who toil below<br>
+            they wait in line and do not complain<br>
+            and finally complete their task<br>
+            at 100% my job is done, and I disappear<br>
```

Comment:
Look into adding a text edit to disappear to give the ending of the poem a stronger feel.